src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -858,4 +858,11 @@ public World getEndWorld() {
         return Bukkit.getWorld(IridiumSkyblock.getInstance().getConfiguration().worldName + "_the_end");
     }
 
+    public boolean isIslandWorld(String world) {
```

Comment:
i was needed this while making an addon to iridium, i was asked why we haven't it, now i make it but tooo late :(